pom.xml

```diff
@@ -111,19 +111,11 @@
   </properties>
 
   <dependencies>
-    <!-- till https://github.com/checkstyle/sonar-checkstyle/issues/40 -->
     <dependency>
-      <groupId>org.sonarsource.java</groupId>
-      <artifactId>sonar-java-plugin</artifactId>
-      <version>${sonar-java.version}</version>
+      <groupId>com.fasterxml.staxmate</groupId>
+      <artifactId>staxmate</artifactId>
```

Comment:
New dependency was required since the other included it and we still do as well.